core/src/jvmTest/kotlin/com/ustadmobile/core/io/UploadSessionTest.kt

```diff
@@ -64,9 +64,6 @@ class UploadSessionTest {
         Napier.baseDebugIfNotEnabled()
```

Comment:
Can't we remove uploadsession and uploadsessionmanager? These are not being used anymore.